providers/aws-s3/src/main/java/org/jclouds/aws/s3/blobstore/AWSS3BlobRequestSigner.java

```diff
@@ -56,7 +56,7 @@
    public AWSS3BlobRequestSigner(RestAnnotationProcessor processor, BlobToObject blobToObject,
          BlobToHttpGetOptions blob2HttpGetOptions, Class<AWSS3Client> interfaceClass,
          @org.jclouds.location.Provider Supplier<Credentials> credentials,
-         RequestAuthorizeSignature authSigner, @TimeStamp Provider<String> timeStampProvider,
+         RequestAuthorizeSignatureV2 authSigner, @TimeStamp Provider<String> timeStampProvider,
```

Comment:
BlobRequestSigner Generates signed requests for blobs. useful in other tools such as backup utilities.
keep AWSS3BlobRequestSigner use RequestAuthorizeSignatureV2 to signed requests for blobs as before.